algorithms/CPlusPlus/Strings/ReverseTheStringWordwise.cpp

```diff
@@ -0,0 +1,44 @@
+// Description :- Given a string, the task is to reverse the order of the words in the given string. 
+// Example :-
+// Input 1:
+//    A = "the sky is blue"
+// Input 2:
+//    A = "this is ib"
+// Output 1:
+//    "blue is sky the"
+// Output 2:
+//    "ib is this"
+
+
+// Time Complexity = O(N), Space Complexity = O(N)
+string Solution::solve(string s) {
+   vector<string>v;
```

Comment:
the program extensively uses objects from standard namespace without explicitly specifying the system about the same. 

either specify that the object belong to standard namespace ( for example `std::vector<std::string> v:`, `std::cin` etc ) or simply tell the system that you are using standard namespace thought the program ( `using namespace std;`)